pkg/kv/kvserver/replica_rangefeed.go

```diff
@@ -484,17 +488,8 @@ func (r *Replica) registerWithRangefeedRaftMuLocked(
 	}
 	r.rangefeedMu.Unlock()
 
-	// Determine if this is a system span, which should get priority.
-	//
-	// TODO(erikgrinaker): With dynamic system tables, this should really check
-	// catalog.IsSystemDescriptor() for the table descriptor, but we don't have
-	// easy access to it here. Consider plumbing this down from the client
-	// instead. See: https://github.com/cockroachdb/cockroach/issues/110883
-	isSystemSpan := span.EndKey.Compare(
-		roachpb.RKey(keys.SystemSQLCodec.TablePrefix(keys.MaxReservedDescID+1))) <= 0
-
 	// Create a new rangefeed.
-	feedBudget := r.store.GetStoreConfig().RangefeedBudgetFactory.CreateBudget(isSystemSpan)
+	feedBudget := r.store.GetStoreConfig().RangefeedBudgetFactory.CreateBudget(isSystemRangefeed)
```

Comment:
nit: should we update the comment here https://github.com/cockroachdb/cockroach/blob/13f8eb8126aa2bb1b2ed76948d1b8dc01b5919ac/pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go#L650-L651 to reflect this change